test/e2e/network_test.go

```diff
@@ -373,7 +374,7 @@ func (tc *testContext) deleteNamespace(name string) error {
 // deployWindowsWebServer creates a deployment with a single Windows Server pod, listening on port 80
 func (tc *testContext) deployWindowsWebServer(name string, affinity *v1.Affinity) (*appsv1.Deployment, error) {
 	// This will run a Server on the container, which can be reached with a GET request
-	winServerCommand := []string{tc.getPowerShellExe(), "-command",
+	winServerCommand := []string{powerShellExe, "-command",
```

Comment:
Consider bringing `pwsh.exe` inline here, instead of introducing a new constant that derails the reading flow with an equivalent value. For maintenance purpose a "_find and replace all_" is good enough, since  `pwsh.exe`  is tightly coupled with `-command`.

Reply:
I would agree if `pwsh.exe` was used only in spot, but given it is used in multiple places it calls for a constant.